src/components/ResultScreen.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable react/forbid-prop-types */
```

Comment:
- [ ] Let's don't disable linter rules. These rules are here to help us write the best code.